MOBILE_CLIENT/COMMON/_HIGH/XPATH_INJECTION/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+    "risk_rating": "high",
+    "short_description": "XPath Injection allows an attacker to modify the intended query of an application, leading to unauthorized access to sensitive data or functionality.",
+    "references": {
+        "OWASP": "https://owasp.org/www-community/attacks/XPATH_Injection",
+        "NIST": "https://nvd.nist.gov/vuln/detail/CVE-2016-6272"
+    },
+    "title": "XPath Injection Vulnerability",
+    "privacy_issue": false,
+    "security_issue": true,
+    "categories": {
+        "OWASP_MASVS_L1": [
+			"MSTG_PLATFORM_2"
+		],
+		"OWASP_MASVS_L2": [
+            "MSTG_PLATFORM_2"
+		],
+        "CWE_TOP_25": [
+            "CWE_89"
+        ],
+        "OWASP_TOP_10": [
+			"A03:2021-Injection"
+		]
+    }
```

Comment:
The formatting of this seems off.

Reply:
[0285859](https://github.com/Ostorlab/KB/pull/58/commits/0285859eaa66eb530bf095a3e3bbf4711eeab044)